registrations/test_views.py

```diff
@@ -111,3 +113,82 @@ def test_get_registration_only_by_user(self):
         response = self.normalclient.get(
             '/api/v1/jembiregistration/{}/'.format(reg.id))
         self.assertEqual(response.status_code, 403)
+
+
+class PositionTrackerViewsetTests(AuthenticatedAPITestCase):
+    def test_authentication_required(self):
+        """
+        Authentication must be provided in order to access the endpoint
+        """
+        url = reverse('positiontracker-list')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 401)
+
+        response = self.normalclient.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    def test_permission_required(self):
+        """
+        The user must have the required permission to be able to perform the
+        requested action
+        """
+        url = reverse('positiontracker-list')
+        response = self.normalclient.post(url, data={'label': 'test'})
+        self.assertEqual(response.status_code, 403)
+        self.assertEqual(PositionTracker.objects.count(), 0)
+
+        self.normaluser.user_permissions.add(
+            Permission.objects.get(name='Can add position tracker'))
+
+        response = self.normalclient.post(url, data={'label': 'test'})
+        self.assertEqual(response.status_code, 201)
```

Comment:
Curious on why is the status_code different here as compared to line 137? Because it looks like the lines leading up to both 137 and line 144 do the same thing. Is it because of the user_permissions.add() in line 140?